src/main/java/com/example/JavaProject/repository/IngredientRepository.java

```diff
@@ -11,5 +11,6 @@ public interface IngredientRepository extends JpaRepository<Ingredient, Long> {
     Ingredient findByIngredientNameAndMeasurement(String ingredient, String measurement);
     Boolean existsById(long id);
```

Comment:
this method is not used, you can remove it

Reply:
Check out newest commit